package.json

```diff
@@ -4,6 +4,11 @@
   "engines": {
     "node": ">=0.10"
   },
+  "files": [
+    "src",
```

Comment:
why do we need src in the artifact?

Reply:
As mentioned in the PR description, without the source step debugging doesn't work as the source the sourcemaps point to cannot be found. The alternative would be inlining sources into the source maps (base64).

Reply:
Another alternative is to avoid publishing both source and source maps.